devdoc/UserBuildAssets.md

```diff
@@ -0,0 +1,140 @@
+# Build Assets for individual developer
+
+## Build Assets
+The modern tool used for generating all our official assets is the Github Actions build-asset workflow on the hpcc-systems/HPCC-Platform repository, located [here](https://github.com/hpcc-systems/HPCC-Platform/actions/workflows/build-assets.yml).  But developers and contributors can utilize this same workflow on their own forked repository. This allows developers to quickly create assets for testing changes and test for build breaks before the peer review process.
+
+Build assets will generate every available project under the HPCC-Platform namespace. There currently is not an option to control which packages in the build matrix get generated.  But most packages get built in parallel, and __released__ after the individual matrix job is completed, so there is no waiting on packages you don't need. Exceptions to this are for packages that require other builds to complete, such as the __ECLIDE__.
```

Comment:
The project list is out of sync with the builds triggered by a PR, and the difference is not just regression and smoke tests. The PR builds newer Ubuntu and macOS targets, at least. If this discrepancy is intentional, it might be worth commenting on it here.

Reply:
This script is the one we use for release builds. So if you build against 9.2.x or 9.4.x you might be behind what the newest PR build distributions are, but the target distributions are pinned to what was available when we cut that major.minor.